project/AppDependencies.scala

```diff
@@ -6,19 +6,21 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-backend-play-28"  % "5.13.0",
+    "uk.gov.hmrc"             %% "bootstrap-backend-play-28"  % "5.20.0",
     "uk.gov.hmrc.mongo"       %% "hmrc-mongo-play-28"         % "0.53.0",
```

Comment:
we can update the mongo lib as well. I think the latest is 0.59 and the cats one 2.7.0

Reply:
Ok, will do.

Reply:
This has been done.